### PR TITLE
Feat/access cleanup

### PR DESCRIPTION
**Pull request by tenthwalker**
What I did:
This pull request updates proptypes throughout the application, fixes an accessibility label issue, adds my contributions to the README reflections, and cleans up comments and console logs.
Did this break anything?
- [x] No
- [ ]  Yes
Type of change
- [x] New feature (non-breaking change which adds functionality)
- [x]  Styling
- [ ]  Bug fix (non-breaking change which fixes an issue)
- [ ]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [x]  Super small fix (Corrected a typo, removed a comment, etc.)
- [ ]  Skip all the other stuff and briefly explain the fix.
Checklist:
- [ ]  If this code needs to be tested, all tests are passing
- [x]  The code runs locally
- [ ]  There are comments where clarification/ organization is needed
- [x]  The code is DRY. If it's not, I tried my best
- [x]  I reviewed my code before pushing
What's next?
Video demo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ NPS Service is an app designed to let you view all of the national parks in the
 ## 💻 Installation instructions
 1. Open project on Github
 2. Clone down the repo
-3. cd into the repo and run `npm install`, `npm i @material-tailwind/react`,
+3. cd into the repo and run `npm install`
```
Comment (Jesuitman) [inline README.md:7]: excellent

File: public/index.html [hunk 2]
```diff
@@ -10,34 +10,11 @@
...
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
     <title>NPS Service</title>
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <div id="root"></div>
-    <!--
```
Comment (Jesuitman) [inline public/index.html:32]: hell yeah!

File: src/Carousel/Carousel.js [hunk 3]
```diff
@@ -12,12 +12,22 @@ export default function NewCarousel({ parkMedia }) {
       alt={image.altText}
     />
   ));
-  console.log(images);
+
```
Comment (Jesuitman) [inline src/Carousel/Carousel.js:15]: good bye console logs New York, hello New York!

File: src/Login/Login.jsx [hunk 4]
```diff
@@ -14,15 +14,13 @@ export default function Login({loggedIn, setLoggedIn}) {
 
   const sendUser = async (user) => {
     try {
-      const response = await axios.get();
+      const response = await axios.get();    //send login info to backend for verification (backend endpoint not complete at time of submission)
```
Comment (Jesuitman) [inline src/Login/Login.jsx:17]: hell yeah!

**Review by Jesuitman — APPROVED** [review]
awesome stuff!

**Review by JoeyStewart — APPROVED** [review]
Nice cleanup here

**Pull request merged**